WebApplication1/Services/RoleService.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using WebApplication1.Configuration;
+
+namespace WebApplication1.Services
+{
+    public class RoleService
+    {
+        private readonly SocketGuild _guild;
+        public readonly Config Config;
+
+        public RoleService(Config config, DiscordSocketClient client)
+        {
+            Config = config;
+            _guild = client.Guilds.FirstOrDefault(x => x.Id == Config.GuildId);
+        }
+
+        public async Task AssignRoleAsync(ulong userId, IEnumerable<string> roles)
+        {
+            // TODO find the nullref bug
```

Comment:
Found it?

Reply:
I think so, couldn't replicate it now, removed the comment